sources/packages/backend/apps/db-migrations/src/sql/Queue/Create-queue-configurations.sql

```diff
@@ -0,0 +1,15 @@
+-- Create new table queue_configurations.
+CREATE TABLE IF NOT EXISTS queue_configurations(
```

Comment:
`sims.queue_configurations ?`

Reply:
it will work without sims. when creating. we are not using `sims` while creating tables

Reply:
I saw that in most of the migrations which create table uses schema. So to be consistent I would say to use schema.

![image](https://user-images.githubusercontent.com/54600590/206051514-df48145b-402c-4dd5-9cda-1d99879aeda4.png)

Reply:
I think I refreshed the old example. adding for the sake of consistency